scripts/frontend/config.js

```diff
@@ -3,6 +3,7 @@ const { resolve } = require('path');
 const root = resolve(__dirname, '..', '..');
 const dist = resolve(root, 'dist');
 const target = resolve(root, 'target');
+const statik = resolve(root, 'src', 'static');
```

Comment:
This is called `statik` because I otherwise get warnings from TypeScript about `static` being a reserved word.

Reply:
🤣